partner_identification/models/res_partner.py

```diff
@@ -19,23 +19,27 @@ class ResPartner(orm.Model):
     The base class provides for the registration of categories of
     identification and a list of those id numbers related to a partner.
 
-    Inheriting modules can dd specific fields, to enable easy entry and search
-    on those fields. An example of souch a field follows here:
+    Inheriting modules can have specific fields, to enable easy entry and
+    search on those fields. An example of souch a field follows here:
 
     Example:
         .. code-block:: python
 
-        _columns = {
-            'social_security': fields.function(
-                _compute_identification,
-                arg='SSN',
-                fcnt_inv=_inverse_identification,
-                fcnt_inv_arg='SSN',
-                type='char',
-                fcnt_search=_search_identification,
-                method= True, store=True,
-            ),
-        }
+    _columns = {
+        'social_security': fields.function(
+            lambda self, *args, **kwargs:
+            self._compute_identification(*args, **kwargs),
+            arg='SSN',
+            fnct_inv=lambda self, *args, **kwargs:
+            self._inverse_identification(*args, **kwargs),
+            fnct_inv_arg='SSN',
+            type='char',
+            fnct_search=lambda self, *args, **kwargs:
+            self._search_identification(*args, **kwargs),
+            method=True, store=True, readonly=False,
```

Comment:
I believe this structurally can't work when the field is stored

Reply:
What do you mean with 'this'?

Reply:
store=True. But as you've shown above, this doesn't apply to 7. Hmmm, that's actually nicer than what newer versions do. But anyways, resolved